plugins/smartvol/bricksplanner/utils.go

```diff
@@ -42,7 +42,8 @@ func getAvailableVgs(req *smartvolapi.Volume) ([]Vg, error) {
 
 	for _, p := range peers {
 		// If Peer is not online, do not consider this device/peer
-		if !store.Store.IsNodeAlive(p.ID) {
+		_, online := store.Store.IsNodeAlive(p.ID)
```

Comment:
Suggestion: You can use the short notation as that variable isn't used elsewhere 

```go
if _, online := store.Store.IsNodeAlive(p.ID); !online {
    ...
}
```